woocommerce/woocommerce
# dev: core profiler pages storybook

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

- Added storybook demos for the core profiler pages
- Tightened up the TS types for the core profiler pages components so that they only ask for what they use



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. run `pnpm --filter=storybook run storybook`
2. Storybook pages for Core Profiler should be present

https://github.com/woocommerce/woocommerce/assets/27843274/13e52943-e15a-40de-8743-1c06dbd923ad




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
